tests/generation/test_utils.py

```diff
@@ -1409,6 +1409,9 @@ def _prepare_model_kwargs(input_ids, attention_mask, signature):
             model = model_class(config).to(torch_device).eval()
             signature = inspect.signature(model.forward).parameters.keys()
 
+            # no cache as some models require special cache classes to be init outside forward
+            config.use_cache = False
```

Comment:
```suggestion
            model.genertation_config.use_cache = False
```
Let's avoid setting `generate` arguments in `config` :)